lib/FusionInventory/Agent/Task/Inventory/Win32/Softwares.pm

```diff
@@ -335,7 +233,7 @@ sub _getSoftwaresList {
         push @list, $software;
     }
 
-    return \@list;
+    return @list;
```

Comment:
The choice to return an array reference was deliberate here, as it was potentially quite long, and the function name is also explicit: _getSoftwaresList shows it returns a unique list, not multiple items. If you prefer to return a list, you'd better rename the function also to _getSoftwares.

Reply:
Okay I agree with you, I'll rollback this adding a comment to clarify the choice. Thank you.